src/content/en/updates/2017/09/chrome-62-media-updates.md

```diff
@@ -0,0 +1,173 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the media (audio/video) updates in Chrome 62.
+
+{# wf_updated_on: 2017-09-08 #}
+{# wf_published_on: 2017-09-08 #}
+{# wf_tags: news,chrome62,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Blink>Media #}
+
+# Media (Audio/Video) Updates in Chrome 62 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- [Offline playback with DRM licenses](#offline-playback) is now supported on
+  Android.
+- Chrome now [disables video tracks when a MSE video is played in the
```

Comment:
It's 'an MSE' not 'a MSE'.

Reply:
I didn't know we should use "an" for acronyms. Thank you @jpmedley!